ide/ide.kit/test/qa-functional/src/org/netbeans/test/ide/CountingSecurityManager.java

```diff
@@ -112,7 +113,7 @@ public static void initialize(String prefix, Mode mode, Set<String> allowedFiles
 
     static void assertReflection(int maxCount, String whitelist) {
         System.setProperty("counting.reflection.whitelist", whitelist);
-        System.getSecurityManager().checkMemberAccess(null, maxCount);
```

Comment:
System.getSecurityManager().checkMemberAccess(null, maxCount)
was removed and not added back using reflection as firstArgument on method is null because of which call on the method in JDK-8 will throw NPE